### PR TITLE
Report the environment in TSV loader Slack notifications

### LINKED ISSUE #381: Report the environment in TSV Slack messages
## Problem

When a TSV load_complete Slack notification is fired, it's impossible to tell what environment it came from. This can cause confusion if someone runs a provider DAG in staging or in their local environment with the `slack_message_override` enabled, because it's not easy to distinguish the notifications.

## Description

Slack notifications sent by the ingestion server are configured to report the [environment](https://github.com/WordPress/openverse-api/blob/main/ingestion_server/ingestion_server/slack.py#L34). We should simply borrow this logic for the Catalog Slack alerting utility, and make sure that it applies to both alerts and notifications.

## Implementation

- [x] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by stacimc**
## Fixes

Fixes #381

## Description

Adds the environment to the username in TSV loader Slack notifications and alerts. This allows us to distinguish between messages sent from production and ones sent from staging or local testing environments.

Screen Shot 2022-02-25 at 10 02 34 AM

## Testing Instructions

`just test`

Set `slack_message_override` environment variable to `true` in your local environment, and then run a provider workflow. Verify that the Slack notification sent on load_complete shows the environment (see screenshot above).

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/common/slack.py [hunk 1]
```diff
@@ -224,7 +224,11 @@ def send_message(
     """Send a simple slack message, convenience message for short/simple messages."""
     if not should_send_message(http_conn_id):
         return
-    s = SlackMessage(username, icon_emoji, http_conn_id=http_conn_id)
+
+    environment = Variable.get("environment", default_var="dev")
```
Comment (stacimc) [inline openverse_catalog/dags/common/slack.py:228]: We are setting the environment to "dev" by default [elsewhere in this file](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/common/slack.py#L245) so I copied it.

[In the API](https://github.com/WordPress/openverse-api/blob/main/ingestion_server/ingestion_server/slack.py#L29) we explicitly use `local`, and I believe `dev` _ought_ to refer to `staging`. Would love some feedback on what to use here in the Catalog 🤔

Comment (AetherUnbound) [reply openverse_catalog/dags/common/slack.py:228]: Ah, yes - this is all over the place :sweat_smile: We have a ticket for unifying them across the board: https://github.com/WordPress/openverse-api/issues/478

For now I think keep the standards that already exist within a file/repo is solid until we tackle that ticket!

**Review by AetherUnbound — APPROVED** [review]
Looks great! Wonderful idea!

**Review by sarayourfriend — APPROVED** [review]
LGTM!

**Pull request merged**